web/src/main/java/org/mskcc/cbio/oncokb/controller/GeneController.java

```diff
@@ -53,6 +55,48 @@ List<Gene> getGene(
         return JsonResultFactory.getGene(new ArrayList<>(genes), fields);
     }
 
+    @RequestMapping(value = "/legacy-api/genomic-indicators")
+    public @ResponseBody
+    List<Evidence> getGenomicIndicators(
+        @RequestParam(value = "entrezGeneId", required = false) List<Integer> entrezGeneIds
+        , @RequestParam(value = "hugoSymbol", required = false) List<String> hugoSymbols
+        , @RequestParam(value = "fields", required = false) String fields
+    ) {
+        if (entrezGeneIds == null && hugoSymbols == null) {
+            return new ArrayList<>();
+        }
+
+        Set<Gene> genes = new HashSet<>();
+        if (entrezGeneIds != null) {
+            for (Integer enterz : entrezGeneIds) {
+                Gene gene = GeneUtils.getGeneByEntrezId(enterz);
+                if (gene != null) {
+                    genes.add(gene);
+                }
+            }
+        }
+        if (hugoSymbols != null) {
+            for (String hugoSymbol : hugoSymbols) {
+                Gene gene = GeneUtils.getGeneByHugoSymbol(hugoSymbol);
+                if (gene != null) {
+                    genes.add(gene);
+                }
+            }
+        }
+        Map<Gene, Set<Evidence>> map = EvidenceUtils.getEvidenceByGenes(genes);
+        List<Evidence> evidenceList = new ArrayList<>();
+
+        for (Map.Entry<Gene, Set<Evidence>> entry : map.entrySet()) {
+            for (Evidence evidence : entry.getValue()) {
+                if (evidence.getEvidenceType() == EvidenceType.GENOMIC_INDICATOR) {
+                    evidenceList.add(evidence);
+                }
+            }
+        }
+
+        return JsonResultFactory.getEvidence(evidenceList, fields);
```

Comment:
I would just only return `id, gene, name, description` fields and remove the `fields` parameter. Seems like too much info for them to see all evidence fields.